backend/fms_core/serializers.py

```diff
@@ -1039,4 +1042,15 @@ def get_sex_concordance(self, instance: SampleIdentity):
     
     def get_identity_matches(self, instance: SampleIdentity):
         matches = SampleIdentityMatch.objects.filter(Q(tested=instance)).all()
-        return SampleIdentityMatchSerializer(matches, many=True).data
\ No newline at end of file
+        return SampleIdentityMatchSerializer(matches, many=True).data
+
+class ProfileSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(read_only=True)
```

Comment:
model fields are there by default. I don't think we need to worry about read only here.